n3fit/src/n3fit/tests/test_fit.py

```diff
@@ -86,7 +86,7 @@ def check_fit_results(
     if regenerate:
         shutil.copy2(new_expgrid_file, old_expgrid_file)
         shutil.copy2(new_json_file, regression_json)
-        raise FileNotFoundError("Regression test regenerated, no files to check against")
+        return
```

Comment:
```suggestion
```

If doing it in this way we might as well let it continue and check that the files are readable.